### PR TITLE
CAL-445 Upgrades to latest wss4j and cxf library versions to match DDF.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Upgrades libraries.

*Note: I believe there is a security permission issue, unrelated to this change, with the `video-app` feature. I will be testing that further and potentially resolving in another PR.*

#### Who is reviewing it? 
@mcalcote @brjeter 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@bdeining
@stustison
#### How should this be tested?
Full regression tests of an installation, in particular testing those modules where `Import-Package` exclusions were added.

#### What are the relevant tickets?

[CAL-445](https://codice.atlassian.net/browse/CAL-445)
[DDF-3920](https://codice.atlassian.net/browse/DDF-3920)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -76,7 +77,7 @@
         <!-- Properties for the frontend-maven-plugin -->
         <node.version>v8.0.0</node.version>
         <yarn.version>v1.2.1</yarn.version>
-        <netty.version>4.1.5.Final</netty.version>
+        <netty.version>4.1.16.Final</netty.version>
```
Comment (jlcsmith) [inline pom.xml]: ❓ curious, why the netty version change?

Comment (coyotesqrl) [reply pom.xml]: To match DDF and get us to closer to a recent version.

Comment (jlcsmith) [reply pom.xml]: Ok, that is probably fine.  We do use netty in the video app but I'd be surprised if a minor version bump would be a problem

**Comment by brjeter** [comment]
@blen-desta and I have hero'd this

**Comment by coyotesqrl** [comment]
Sidestepping CI build as manual heroing has verified.

**Pull request merged**